PhD Thesis/Thesis.tex

```diff
@@ -15,8 +15,7 @@
 %	PACKAGES AND OTHER DOCUMENT CONFIGURATIONS (do not modify)
 %-------------------------------------------------------------------------------
 
-\documentclass[12pt, oneside]{book} % 12 pt font, one-sided book style
-\usepackage[a4paper, includehead, headheight=0.6cm, inner=2.5cm ,outer=2.5cm, top=2.5 cm, bottom=2.5cm]{geometry}  % Changing size of document
+\documentclass[final]{oist_thesis} % use \documentclass[final]{oist_thesis} for a final version
```

Comment:
Maybe the comment should read:
> use \documentclass{oist_thesis} for a temporary version for thesis revision and examination

Reply:
Thank you for pointing out the inconsistencies. Actually, I was not sure which part should be modified. As a result, I decided to somehow make a PR, and to ask Jeremie to check changes and modify tex files appropriately.

Reply:
I think maybe we could have both:
```LaTeX
\documentclass[temporary]{oist_thesis} % Temporary version
%\documentclass[final]{oist_thesis} % Final version
```